### ISSUE TITLE
Cannot run emake.exe without MSYS2 environment

### ISSUE THREAD

**Issue by LiarOnce** [enigma-dev/enigma-dev#1434]
I tried to run directly in an environment without MSYS2, but without success, I got a `0xc000007b` error code.What's missing is it?

It works fine in MSYS2

**Comment by RobertBColton** [comment]
That is correct and intentional, it is dynamically linking to boost. I've been a bit slow this week due to a death in the family, my apologies.

I was last working on a commit to static link everything for emake on Windows, but that's proving to be more difficult because I guess the MSYS2 protobuf and grpc packages don't come with a static build. If you really need a static emake, building it with Microsoft's VCPKG, like the RGM repo does, is a viable alternative.
https://github.com/enigma-dev/RadialGM/blob/73d6e5a1349470404d107a505cdc914ca0d765aa/appveyor.yml#L38

Thanks to fundies, all of the front end parts now support Cmake & MSVC, just not the actual engine yet. Later the engine too can be made to support Cmake as well as MSVC, but we're focusing on more important things first.

I will update this here if I discover anything when I get back to that, working on an RGM commit now. The reason I am trying to make emake static is so that it's easier to install RGM when we do the beta, but we also have some other plans too that I am not discussing just yet.

**Comment by LiarOnce** [comment]
Thanks a lot.
